tools/coqdep/lib/options.mli

```diff
@@ -8,4 +8,12 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
+module Dynlink : sig
+  type t = Opt | Byte | Both | No | Variable
+end
+
 val boot : bool ref
+val sort : bool ref
+val write_vos : bool ref
+val noglob : bool ref
+val dynlink : Dynlink.t ref
```

Comment:
At some point in a future PR we could remove these and have the clients use `Args.t` directly.

Reply:
@ejgallego They can initialize Coqdep.Lib with Args.t directly, thats what init does now. These options are simply the internal state of coqdep. We could probably move this to a common state file.